pxtlib/emitter/emitter.ts

```diff
@@ -1367,7 +1367,7 @@ ${lbl}: .short 0xffff
                     }
                     ev = val + ""
                 }
-                if (/^\d+$/.test(ev))
+                if (/^[-]?\d+$/.test(ev))
```

Comment:
The _ev_ variable is set either through checker.getConstantValue or parseComments. The checker.getConstantValue function returns a number so this can't be "+". The parseComments function can return any string, I can't see it setting it to anything other than undefined in the current code path. I am going to fix this to make [+-] as there a future proof.